.travis.yml

```diff
@@ -5,15 +5,12 @@
 # https://puppet.com/docs/pe/2018.1/overview/getting_support_for_pe.html
 # ------------------------------------------------------------------------------
 # Release       Puppet   Ruby   EOL
-# SIMP 6.2      4.10     2.1.9  TBD
-# PE 2016.4     4.10     2.1.9  2018-12-31 (LTS)
```

Comment:
How far back should we be leaving the information? I thought it was always kept for history sake, just like the entirety of the CHANGELOG

Reply:
@sharkbruhaha Meh, I've been removing it when we no longer support it, but it's not a big deal either way since it will eventually be rewritten with a static asset push.